Makefile

```diff
@@ -530,7 +530,7 @@ lint-html:
 	node scripts/html-lint.js
 
 lint-spell:
-	codespell.py \
+	codespell.py --dictionary - --dictionary dictionary \
```

Comment:
I'm not sure I'm following what `--dictionary -` is for here?
Unrelated nit: What do you think about renaming the dictionary file to `dictionary.txt` to avoid confusion about what it is?

Reply:
Its used to load the default dictionary (see https://github.com/lucasdemarchi/codespell/blob/master/codespell_lib/_codespell.py#L547).

> Unrelated nit: What do you think about renaming the dictionary file to dictionary.txt to avoid confusion about what it is?

Sounds good, I'll handle it

Reply:
Haha, mildly cryptic and undocumented